### ISSUE TITLE
Settings: Missing focus ring around section tabs

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#5278]
https://forums.zotero.org/discussion/124130/tab-focus-box-in-settings-sections

**Comment by dstillman** [comment]
There's also the Firefox option, which solves this problem, though I like our background color more:

![Image](https://github.com/user-attachments/assets/4205ab2d-c4e6-49ea-85cb-273545a2f2ea)

![Image](https://github.com/user-attachments/assets/4664d063-4c6d-4de6-9f4c-39d4034bc47b)

So maybe try the gray/blue option?
